client/package.json

```diff
@@ -97,4 +97,4 @@
       "last 1 safari version"
     ]
   }
-}
```

Comment:
I'm not sure if you meant to add this, but when bumping the version we just want to bump [this](https://github.com/COVID-19-electronic-health-system/Corona-tracker/blob/cf3351f2955ebd2981b5d633d9afcf9b7647c5a7/client/package.json#L3) line, 0.2.7->0.2.8 👍

Reply:
Done